3_visualize.R

```diff
@@ -16,6 +19,44 @@ p3_targets_list <- list(
              format = "file",
              pattern = map(p2_inst_data_hourly)),
   
+  # Create and save log file containing data availability summary
+  tar_target(
+    p3_sitelist_summary_csv,
+    summarize_site_list(p2_site_list_csv,p1_daily_data,p1_inst_data,fileout = "3_visualize/log/sitelist_summary.csv"),
+    format = "file"),
```

Comment:
To me this seems like this log file should be in the same directory as the actual file, so `2_process/log/`. I wouldn't think to look in the `visualization` directory for a log file that describes a target build in `2_process.R`. What do you think of the idea of moving it to `2_process/log`

Reply:
I initially put them all in the same `3_visualize/log` directory because I thought it might make reviewing/referencing them easier if they were all in one place. But I definitely see your point (and have wondered the same thing myself). If it'd be more intuitive, I can re-arrange the log files into phase-specific log folders.

Reply:
Ah. I see. In the pipelines I've seen they are in the same "number" directory as the target itself: https://github.com/USGS-R/delaware-model-prep/tree/main/2_observations/out

Reply:
OK, that makes sense - thanks for linking to that example. I've also created a separate log subdirectory based on an example repo in one of the targets trainings. Do you have any preferences for sticking with /log folders vs. writing files to /out as in delaware-model-prep?

Reply:
I'd be fine either way. Slight preference to a `log` sub-directory instead of `out` to keep things a little cleaner in `out`

Reply:
OK, I agree with that too. I'll go ahead and separate out those files, but keep the log folders within individual phases.